slsc-face-archive.md

```diff
@@ -0,0 +1,15 @@
+# Project SLSC Photo Archive Face Tagging
+
+## Client Profile
+
+Youth Exploring Science (YES) program is created to help underserved students in St. Louis area. Students participate in YES program to gain skills in Science, Technology, Engineering, Arts and Mathematics fields. High school students will gain real-world skills, career readiness training through YES program. Students in the YES program also promote the importance of STEAM-related careers.
+
+## Project Brief
+
+The SLSC Face Archive Project captures and preserves various facial expressions and emotions, assisting researchers, psychologists, and developers in disciplines such as mental health, AI, and emotion identification. It adheres to ethical norms, ensuring participant consent, privacy, and anonymity, and has a wide range of applications, including investigating emotional responses and refining AI algorithms.
+
+## Functional Requirements
+
+* Data(All the photos with their name tags),
+* As an admin, They can upload, edit and delete the images and tag them with their respective names.
+* As a User, he/she should be able to verify whether to accept or reject the tag.
```

Comment:
Data collection.
Preprocess data.
Train a model/ Adapt a pre-trained model to fit our data.
Build UI/ Integrate with existing UI.

Reply:
incorporated comments into updates.